frontend/elements/src/core/titles/variants/plain-white.ts

```diff
@@ -24,13 +28,15 @@ export class _ extends LitElement {
   title:string = ""; 
   @property({type: Boolean})
   bold:boolean = false; 
+  @property({type: Boolean})
+  small:boolean = false; 
 
   render() {
 
-    const {title, bold} = this;
+    const {title, bold,small} = this;
 
     return html`
-    <p class="${bold ? 'bold' : ''}">${title}</p>
+    <p class="${bold ? 'bold' : ''} ${small ? 'small' : ''}">${title}</p>
```

Comment:
This file shouldn't be used anywhere, instead we have `title-ji` with color options. Please delete these temporary variant references when you've moved it over. If the differences can't be expressed in `title-ji` then we may need new elements, but color alone isn't a problem